WordPress/src/main/java/org/wordpress/android/ui/reader/discover/ReaderDiscoverViewModel.kt

```diff
@@ -218,8 +228,10 @@ class ReaderDiscoverViewModel @Inject constructor(
     }
 
     private fun onTagItemClicked(tagSlug: String) {
-        val readerTag = readerUtilsWrapper.getTagFromTagName(tagSlug, FOLLOWED)
-        _navigationEvents.postValue(Event(ShowPostsByTag(readerTag)))
+        launch(bgDispatcher) {
+            val readerTag = readerUtilsWrapper.getTagFromTagName(tagSlug, FOLLOWED)
```

Comment:
Since getTagFromTagName isn't a suspendable function I decided to start this coroutine on a bg thread.

Reply:
Same comment as earlier. Use ioDispatcher or bg?

Reply:
Same as earlier comment.

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/12806/commits/c4648919c2747c18707d58804c4a974bffbf5496